### PR TITLE
Make finish page properly handle vscode settings and popup

### LINKED ISSUE #20: "Finished: Use desktop to Open VS Code" is displayed regardless of what was installed
If you install just Java and the tools, after installation is complete, it displays "Finished: Use desktop to Open VS Code". The prompt should change if VS Code isn't installed.

### LINKED ISSUE #43: Open VS Code directory on macOS after installation
`open /path/to/folder` will open the folder in Finder.

### PR DESCRIPTION
**Pull request by ThadHouse**
Closes #20  and Closes #43

### CODE REVIEW

File: WPILibInstaller-Avalonia/ViewModels/FinalPageViewModel.cs [hunk 1]
```diff
@@ -1,19 +1,53 @@
...
+                    FinishText = "Finish will open the VS Code folder. Please drag this to your dock.";
+                }
+                else
+                {
+                    FinishText = "Use the WPILib VS Code desktop icon to start VS Code.";
+                }
+            }
+            else
+            {
+                FinishText = "";
```
Comment (sciencewhiz) [inline WPILibInstaller-Avalonia/ViewModels/FinalPageViewModel.cs:33]: How about successfully installed, or something like that?

File: WPILibInstaller-Avalonia/ViewModels/FinalPageViewModel.cs [hunk 2]
```diff
@@ -1,19 +1,53 @@
...
+
+        public FinalPageViewModel(IProgramWindow progWindow, IConfigurationProvider configurationProvider, IToInstallProvider toInstallProvider)
             : base("Finish", "")
         {
+            vsCodeInstalled = toInstallProvider.Model.InstallVsCode;
+            if (vsCodeInstalled)
+            {
+                if (OperatingSystem.IsMacOS())
+                {
+                    FinishText = "Finish will open the VS Code folder. Please drag this to your dock.";
```
Comment (sciencewhiz) [inline WPILibInstaller-Avalonia/ViewModels/FinalPageViewModel.cs:24]: How about clicking Finish, or something like that?

File: WPILibInstaller-Avalonia/ViewModels/FinalPageViewModel.cs (hunk 1 above)
Comment (ThadHouse) [reply WPILibInstaller-Avalonia/ViewModels/FinalPageViewModel.cs:33]: Thats already part of the window, on a separate line.

**Comment by AustinShalit** [comment]
Instead of asking the user to drag the files into the dock, can we do it for them by editing the com.apple.dock.plist? Seems like we could add an entry under the persistent-others array and reboot the dock to make it work.

**Comment by ThadHouse** [comment]
Likely not in this release. If you can figure out a way to do that from C# sure.

**Pull request merged**